### PR TITLE
[wicket] Add sled+switch location to status bar

### LINKED ISSUE #3777: [wicket] identify where wicket is running on
It would be nice if wicket would tell me where it was running.

I have two switches and sometimes it's not obvious which one I ran wicket on.
If the serial number of the system where wicket is displaying from was shown or available
somewhere, it would be nice.

### PR DESCRIPTION
**Pull request by jgallagher**
I don't think `Location:` is the right label for this - open to suggestions. Maybe "Connected to:"? Example from madrid:

wicket-statusbar

Fixes #3777

### CODE REVIEW

**Comment by citrus-it** [comment]
> I don't think `Location:` is the right label for this - open to suggestions. Maybe "Connected to:"? Example from madrid:

`Ident:`?

**Comment by luqmana** [comment]
or `(Connected) via`?

**Review by andrewjstone — APPROVED** [review]
LGTM. I have no better suggestions than `Location`

**Comment by jgallagher** [comment]
How about `You are here:`? (h/t @mkeeter)

File: wicketd/Cargo.toml [hunk 1]
```diff
@@ -20,6 +20,7 @@ gateway-messages.workspace = true
 hex.workspace = true
 http.workspace = true
 hyper.workspace = true
+once_cell.workspace = true
```
[the remark this answers is not in the document]
Comment (jgallagher) [reply wicketd/Cargo.toml]: Ah, thanks. I searched for `OnceCell` and only saw the non-`sync` version; I didn't realize it got renamed to `OnceLock` in `std`.

**Review by sunshowers — APPROVED** [review]
Looks great!

**Pull request merged**